content/en/docs/instrumentation/net/automatic/annotations.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Annotations
```

Comment:
For sure not annotations. .NET does not have it (look like copy-issue from JAva ;) ). 

It is more like extending automation with manual traces and metrics

Reply:
+1!